src/app/[orgSlug]/study/[studyId]/resubmit/form.tsx

```diff
@@ -37,6 +37,8 @@ export function ResubmitStudyCodeForm(props: { study: SelectedStudy }) {
         },
     })
 
+    const orgSlug = study.submittedByOrgSlug || study.orgSlug
```

Comment:
this is `resubmit` so we should always have a `submittedByOrgSlug`.  I don't think we should use orgSlug as fallback, let's throw an error.

```suggestion
    const orgSlug = study.submittedByOrgSlug
    if (!orgSlug) throw Error('no submitted by org for study')
```